fcrepo-http-api/src/main/java/org/fcrepo/http/api/FedoraLdp.java

```diff
@@ -581,10 +581,16 @@ public Response createOrReplaceObjectRdf(
                 }
 
             LOGGER.debug("Finished creating resource with path: {}", externalPath());
-
             return createUpdateResponse(getFedoraResource(transaction, fedoraId), created.get());
         } finally {
             transaction.releaseResourceLocksIfShortLived();
+            if (requestBodyStream != null) {
```

Comment:
We could use IOUtils.closeQuietly to close these streams since it'd do the null check and doesn't care if there's an error (I don't think we care), which would also avoid the codecov complaints about coverage.